workflows/phaseimpute/tests/test_all.nf.test.snap

```diff
@@ -0,0 +1,250 @@
+                "prep_panel/chunks/glimpse1/1000GP.s.norel_chr22_chunks_glimpse1.txt",
+                "prep_panel/chunks/glimpse1/versions.yml",
+                "prep_panel/chunks/glimpse2/1000GP.s.norel_chr21_chunks_glimpse2.txt",
+                "prep_panel/chunks/glimpse2/1000GP.s.norel_chr22_chunks_glimpse2.txt",
+                "prep_panel/chunks/glimpse2/versions.yml",
+                "prep_panel/haplegend/1000GP.s.norel_chr21.hap.gz",
+                "prep_panel/haplegend/1000GP.s.norel_chr21.legend.gz",
+                "prep_panel/haplegend/1000GP.s.norel_chr21.samples",
+                "prep_panel/haplegend/1000GP.s.norel_chr22.hap.gz",
+                "prep_panel/haplegend/1000GP.s.norel_chr22.legend.gz",
+                "prep_panel/haplegend/1000GP.s.norel_chr22.samples",
+                "prep_panel/haplegend/versions.yml",
+                "prep_panel/posfile/1000GP.s.norel_chr21_posfile_stitch.txt",
+                "prep_panel/posfile/1000GP.s.norel_chr22_posfile_stitch.txt",
+                "prep_panel/posfile/versions.yml",
+                "prep_panel/sites/tsv/1000GP.s.norel_chr21_glimpse1_sites_tsv.txt.gz",
+                "prep_panel/sites/tsv/1000GP.s.norel_chr21_glimpse1_sites_tsv.txt.gz.tbi",
+                "prep_panel/sites/tsv/1000GP.s.norel_chr22_glimpse1_sites_tsv.txt.gz",
+                "prep_panel/sites/tsv/1000GP.s.norel_chr22_glimpse1_sites_tsv.txt.gz.tbi",
+                "prep_panel/sites/tsv/versions.yml",
+                "prep_panel/sites/vcf/1000GP.s.norel_chr21_glimpse1_sites.vcf.gz",
+                "prep_panel/sites/vcf/1000GP.s.norel_chr21_glimpse1_sites.vcf.gz.csi",
+                "prep_panel/sites/vcf/1000GP.s.norel_chr22_glimpse1_sites.vcf.gz",
+                "prep_panel/sites/vcf/1000GP.s.norel_chr22_glimpse1_sites.vcf.gz.csi",
+                "prep_panel/sites/vcf/versions.yml"
+            ],
+            [
+                "validation/NA12878_Dnull_P1000GP.s.norel_Call_SNP.txt",
+                "validation/NA19401_Dnull_P1000GP.s.norel_Call_SNP.txt",
+                "validation/NA20359_Dnull_P1000GP.s.norel_Call_SNP.txt",
+                "validation/TestQuality.txt",
+                "validation/concat/NA12878_truth_concat.vcf.gz",
+                "validation/concat/NA12878_truth_concat.vcf.gz.tbi",
+                "validation/concat/NA19401_truth_concat.vcf.gz",
+                "validation/concat/NA19401_truth_concat.vcf.gz.tbi",
+                "validation/concat/NA20359_truth_concat.vcf.gz",
+                "validation/concat/NA20359_truth_concat.vcf.gz.tbi"
+            ],
+            1779
+        ],
+        "meta": {
+            "nf-test": "0.8.4",
+            "nextflow": "23.10.1"
+        },
+        "timestamp": "2024-05-17T21:48:07.027867847"
+    },
+    "Check test_validate": {
+        "content": [
+            [
+                "validation/NA12878_Dnull_P1000GP.s.norel_Cchr21_SNP.txt",
```

Comment:
Should validation be per chromosome? Do we want a whole vcf validation as well?

Reply:
This shouldn't be done by chromosomes.

Reply:
I'll make the appropriate modifications.